ORK1Kit/ORK1Kit/Common/ORK1Step.h

```diff
@@ -233,6 +233,13 @@ The default value of this property is 'NO'.
 */
 @property (nonatomic, assign) BOOL excludeFromProgressCalculation;
 
+/**
+An override title for the next/done button.
+
+The default value of this property is 'nil'.
+*/
+@property (nonatomic, copy, nullable) NSString *overrideNextButtonText;
```

Comment:
Can this just be named `nextButtonText` instead?

Reply:
👍

Reply:
Latest changes LGTM